lib/reducers/create-otp-reducer.js

```diff
@@ -142,9 +143,21 @@ export function getInitialState(userDefinedConfig) {
   if (currentQuery.routingType === 'ITINERARY') {
     queryModes = ensureSingleAccessMode(queryModes)
   }
-  if (config.modes && config.modes.combinations) {
+  if (config.modes?.combinations) {
```

Comment:
```suggestion
  if (config?.modes?.combinations) {
```
I think this missing question mark is causing the crash